core/trino-main/src/main/java/io/trino/server/ui/OAuth2WebUiAuthenticationFilter.java

```diff
@@ -137,7 +139,14 @@ private void needAuthentication(ContainerRequestContext request)
             sendWwwAuthenticate(request, "Unauthorized", ImmutableSet.of(TRINO_FORM_LOGIN));
             return;
         }
-        request.abortWith(service.startOAuth2Challenge(request.getUriInfo()));
+        request.abortWith(startOAuth2Challenge(request.getUriInfo()));
```

Comment:
Inline since this is only used in one place

Reply:
Done.